types/qrcode-svg/index.d.ts

```diff
@@ -0,0 +1,107 @@
+// Type definitions for qrcode-svg 1.0
+// Project: https://github.com/papnkukn/qrcode-svg
+// Definitions by: Eric Ferreira <https://github.com/ericbf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Generate an svg QR code easily with this class. Just instatiate an instance
+ * with one of the constructors, then either generate the QR code svg as a
+ * string with `.svg(...)` or save it to a file with `.save(...)`.
+ */
+declare class QRCode {
+	/**
+	 * Create a QRCode with the default options and the passed content.
+	 * @param content The content to encode in the QRCode.
+	 */
+	constructor(content: string);
+	/**
+	 * Create a QRCode by specifying its options. Any values not passed in the
+	 *   options object will be inferred as the defaults.
+	 * @param options The options with which to create the QRCode.
+	 */
+	// tslint:disable-next-line:unified-signatures
+	constructor(options: QRCode.Options);
+
+	/** The raw model of this QRCode. */
+    qrcode: QRCode.Model;
+
+	/** The options that were used to create this QRCode. */
+    options: QRCode.Options;
+
+	/**
+	 * Generates an SVG image of this QRCode
+	 * @param  opt Set the container. Defaults to `{ container: "svg" }`.
+	 * @return The svg string.
+	 */
+	svg(opt?: {
+		container: "svg" | "g" | "none"
+	}): string;
+	/**
+	 * Writes this QRCode to a file. Requires `fs`.
+	 * @param  file The filename to write to
+	 * @param  callback The callback that will be called, possibly with an error, when done.
+	 */
+	save(file: string, callback: (error?: Error) => void): void;
+}
+
+declare namespace QRCode {
+    interface Options {
+        /** Default is `"#ffffff"`. */
+        background?: string;
+        /** Default is `"#000000"`. */
+        color?: string;
+        /** The actual content of the QR code */
+        content: string;
+        /** Default is `"M"`. */
+        ecl?: "L" | "M" | "H" | "Q";
+        /** Default is `256`. */
+        height?: number;
+        /** Default is `4`. */
+        padding?: number;
+        /** Default is `256`. */
+        width?: number;
+    }
+
+    interface Model {
+		/**
+		 * This QRCode's data matrix. A square, two-dimensional boolean array
+		 * of `moduleCount` x `moduleCount` size. True in a given slot means
+		 * that spot is filled in in the QRCode visualization.
```

Comment:
This file seems to be mixing tabs and spaces.

Reply:
Fixed